src/jit/codegencommon.cpp

```diff
@@ -1911,6 +1904,11 @@ bool                CodeGen::genCreateAddrMode(GenTreePtr    addr,
     }
 #endif // LEGACY_BACKEND
 
+    /* We come back to 'AGAIN' if we have an add of a constant, and we are folding that
+       constant, or we have gone through a GT_NOP or GT_COMMA node. We never come back
+       here if we find a scaled index.
+    */
+
     /* Special case: keep constants as 'op2' */
```

Comment:
Move this above AGAIN:?

---
In reply to: [72541449](https://github.com/dotnet/coreclr/pull/6491#discussion_r72541449) [](ancestors = 72541449)